CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 Change Log
 ==========
 
+[v0.5.0-c1][] (2017-07-24)
+--------------------------
+
+bug fixes and improvements, make it more stable.
+
+- Bugs
+    + fixed the problem on windows with chocolate and bash-completion.
+    + upgraded and pin down with newest chef version.
+    + fixed problem with rsync_args (thanks to @smxsm).
+
+- Improvements
```

Comment:
btw, we got these features here, it's not the improvements. Please see the type: to know which is the bug, improvement, task and feature.